Timestamp when instead of at which (#1137)
Rebilly/api-definitions: WRITING-STYLE.md

```yaml
  createdTime:

>     description: Date and time when the coupon is created.

Reviewer: ```suggestion
    description: Date and time the coupon was created.
```

Reviewer: Reason: the shortest possible definition, without loss of meaning.

Author: This suggestion violates the future-past rule.

Reviewer: Got it, thank you

```

Coupon expired time:

```yaml
  expiredTime:

>     description: Date and time when the coupon expires.

Reviewer: I also think original property `expiredTime` carries the wrong meaning. The name of the property implies the past tense. We should generally reconsider the naming of properties of this type. Possible good name: `expirationSheduledAt`.

>     description: Date and time when the coupon expires.

Author: Property names for date-time are an issue. But unless we version or deprecate it is going to be difficult to change property names.
